CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -497,4 +512,26 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController *)previewingContext:(id <UIViewControllerPreviewing>)previewingContext viewControllerForLocation:(CGPoint)location {
```

Comment:
I understand this _is_ possible with Objective-C, but I'm just wondering why I should remove it.